content/zh-cn/docs/reference/kubernetes-api/cluster-resources/runtime-class-v1.md

```diff
@@ -0,0 +1,588 @@
+    
+    PodFixed represents the fixed resource overhead associated with running a pod.
+  -->
+  - **overhead.podFixed** (map[string]<a href="{{< ref "../common-definitions/quantity#Quantity" >}}">Quantity</a>)
+
+    podFixed 表示与运行一个 Pod 所关联的资源开销。
+
+<!--
+- **scheduling** (Scheduling)
+  Scheduling holds the scheduling constraints to ensure that pods running with this RuntimeClass are scheduled to nodes that support it. If scheduling is nil, this RuntimeClass is assumed to be supported by all nodes.
+
+  <a name="Scheduling"></a>
+  *Scheduling specifies the scheduling constraints for nodes supporting a RuntimeClass.*
+-->
+- **scheduling** (Scheduling)
+
+  scheduling 包含调度约束，这些约束用来确保以这个 RuntimeClass 运行的 Pod 被调度到支持此运行时类的节点。
+  如果 scheduling 设为空，则假定所有节点支持此 RuntimeClass。
+
+  <a name="Scheduling"></a>
+  **Scheduling 指定支持 RuntimeClass 的节点的调度约束。**
+
+  <!--
+  - **scheduling.nodeSelector** (map[string]string)
+    nodeSelector lists labels that must be present on nodes that support this RuntimeClass. Pods using this RuntimeClass can only be scheduled to a node matched by this selector. The RuntimeClass nodeSelector is merged with a pod's existing nodeSelector. Any conflicts will cause the pod to be rejected in admission.
+  -->
+  - **scheduling.nodeSelector** (map[string]string)
+
+    nodeSelector 列出支持此 RuntimeClass 的节点上必须存在的标签。
+    使用此 RuntimeClass 的 Pod 只能调度到与这个选择算符匹配的节点上。
+    RuntimeClass nodeSelector 与 Pod 现有的 nodeSelector 合并。
+    任何冲突均会使得该 Pod 在准入时被拒绝。
+  
+  <!--
+  - **scheduling.tolerations** ([]Toleration)
+    *Atomic: will be replaced during a merge*
+    
+    tolerations are appended (excluding duplicates) to pods running with this RuntimeClass during admission, effectively unioning the set of nodes tolerated by the pod and the RuntimeClass.
+
+    <a name="Toleration"></a>
+    *The pod this Toleration is attached to tolerates any taint that matches the triple <key,value,effect> using the matching operator <operator>.*
+  -->
+  - **scheduling.tolerations** ([]Toleration)
+
+    **原子性：将在合并期间被替换**
+    
+    tolerations 在准入期间追加到以此 RuntimeClass 运行的 Pod（不包括重复项）上，
+    本质上是求取 Pod 和 RuntimeClass 所容忍的节点并集。
+
+    <a name="Toleration"></a>
+    **附加此容忍度的 Pod 将容忍用匹配运算符 `operator` 运算后与三元组
+    `<key,value,effect>` 匹配的任何污点。**
+
+    <!--
+    - **scheduling.tolerations.key** (string)
+      Key is the taint key that the toleration applies to. Empty means match all taint keys. If the key is empty, operator must be Exists; this combination means to match all values and all keys.
+
+    - **scheduling.tolerations.operator** (string)
+      Operator represents a key's relationship to the value. Valid operators are Exists and Equal. Defaults to Equal. Exists is equivalent to wildcard for value, so that a pod can tolerate all taints of a particular category.
+    -->
+    - **scheduling.tolerations.key** (string)
+
+      key 是容忍度所应用到的污点键。空意味着匹配所有污点键。
+      如果键为空，则运算符必须为 Exists；这个组合意味着匹配所有值和所有键。
```

Comment:
Maybe we have to avoid 4 spaces (not sure why and how).
The preview is broken here.